### ISSUE TITLE
conference videos?

### ISSUE THREAD

**Issue by garryyao** [jsconfcn/ShenJS#32]
I think we will have video recordings for each speech, correct me if I am wrong? Do you have any idea when these videos would be available?

**Comment by quentinberder** [comment]
@garryyao yes we do record all of the videos. We are currently in post production, no firm delivery date, we should however know shortly. I will post it here.

**Comment by fraserxu** [comment]
I checked with the video crew, We will probably have one or two up by tomorrow.

**Comment by hunvreus** [comment]
Would probably be a good time to start thinking about redoing the main JSconf.cn website with the videos, no?

**Comment by mafintosh** [comment]
Looking forward to the videos \o/

**Comment by fraserxu** [comment]
@garryyao I sorted out most of the slide today here in #30 and also have the first sample of the video http://pan.baidu.com/s/1mgzPPOG 

There's some issue with the audio and the sync between the audio and video, I'm asking the video guy to fix and push a new version later today.

Also I sent a post on weibo about the ShenJS Wrapup this Wednesday at EF labs, hope it would help http://weibo.com/jsconfchina

**Comment by garryyao** [comment]
Thanks for the update, first sample video composing looks pretty good, and welcome to join us in the wrapup event to enjoy the videos.

**Comment by hunvreus** [comment]
You mean it looks friggin awesome! I'm bummed I couldn't be there, goddammit!

![Awesome](http://media.giphy.com/media/90F8aUepslB84/giphy.gif)

**Comment by bencrox** [comment]
Hi Fraser and co.

Shall I download from baidu pan and update to youtube or vimeo ?
It loads very slow in HK. YouKu is not better...

regards

Ben

On Mon, Jul 27, 2015 at 6:49 PM, Fraser Xu notifications@github.com wrote:

> @garryyao https://github.com/garryyao I sorted out most of the slide
> today here in #30 https://github.com/jsconfcn/ShenJS/issues/30 and also
> have the first sample of the video http://pan.baidu.com/s/1mgzPPOG
> 
> There's some issue with the audio and the sync between the audio and
> video, I'm asking the video guy to fix and push a new version later today.
> 
> Also I sent a post on weibo about the ShenJS Wrapup this Wednesday at EF
> labs, hope it would help http://weibo.com/jsconfchina
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/jsconfcn/ShenJS/issues/32#issuecomment-125163634.

## 

Ben Lee, +852 3184 0680
Managing Director, Web Fusion Technology Limited

**Comment by fraserxu** [comment]
@bencrox Here you go http://pan.baidu.com/s/1o6CDskq

It includes the first video and all the photo, take whatever you want and thanks!

**Comment by garryyao** [comment]
Is there any blockers with the other videos? May I know from the schedule are the rest of videos being released one by one that we can set an expectation for that?

**Comment by fraserxu** [comment]
Yes, @garryyao sorry for the delay. Actually I've asked the guy serval times but he seems a little busy recently. I'll ping him again and if he's still not available, I'll probably take the raw video and do it by myself.

**Comment by Saviio** [comment]
Hi @fraserxu ,

There's only one video on BaiduPan till now, May I ask when we can see the rest of videos?

**Comment by fraserxu** [comment]
@Saviio I will check with the video crew and have him to upload the other videos later.

**Comment by ajfisher** [comment]
@fraserxu Is there any update on when those vids will be available?

**Comment by Katierichard** [comment]
Link to download the videos -  http://pan.baidu.com/s/1i30Iprn#0-tsina-1-16874-397232819ff9a47a7b7e80a40613cfe1path=%252FShenJS%2520Video&path=%252FShenJS%2520Video

**Comment by Katierichard** [comment]
The videos are now all online on [youtube](https://www.youtube.com/channel/UCvFAekDfG_dL2Kcbq-vrPXQ) and [tudou](http://www.tudou.com/home/wiredcraft).
